angular-primeng-app/src/app/services/blog.service.ts

```diff
@@ -102,23 +106,24 @@ export class BlogService {
 			);
 	}
 
-	getPostsInSeries(host: string, slug: string): Observable<Post[]> {
-		return this.apollo
-			.watchQuery<any>({
-				query: GET_POSTS_IN_SERIES,
-				variables: {
-					host: host,
-					slug: slug,
-				},
-			})
-			.valueChanges.pipe(
-				map(({ data }) =>
-					data.publication.series.posts.edges.map(
-						(edge: { node: any }) => edge.node
-					)
-				)
-			);
-	}
+  getPostsInSeries(host: string, slug: string, after: string = ""): Observable<PostsPageInfo> {
+    return this.apollo
+      .watchQuery<any>({
+        query: GET_POSTS_IN_SERIES,
+        variables: {
+          host: host,
+          slug: slug,
+          after: after
+        },
+      }).valueChanges.pipe(
+        map(({ data }) => {
+          const { edges, pageInfo } = data.publication.series.posts;
+          return {
+            posts: edges.map((edge: { node: any; }) => edge.node),
+            pagination: pageInfo
+          };
+        }));
+  }
```

Comment:
@czBalazs98 everything looks good will be checking it out,
thank you very much for correcting the indentation over here 
- this is the way it's supposed to be :)